src/App.js

```diff
@@ -1,25 +1,21 @@
 import React, { Component } from "react";
 import "./App.css";
-import Navbar from "./Components/NavBar/NavBar";
-import Footer from "./Components/Footer/Footer";
-import LandingPage from "./Components/LandingPage/LandingPage";
-import styled from "@emotion/styled";
+//import Footer from "./Components/Footer/Footer";
+import LandingPage from "./Components/Body/LandingPage";
+//import styled from "@emotion/styled";
 
-const PageContainer = styled.body`
-  width: 100%;
-  height: 100%;
-  background: #ffffff;
-`;
+// const PageContainer = styled.body`
+//   width: 100%;
+//   height: 100%;
+//   background: #ffffff;
+// `;
 
 class App extends Component {
   render() {
-    //const { lightStatus } = this.state;
-
     return (
-      <PageContainer>
+      <body>
```

Comment:
On App.js I would change the body tag to just a div. In HTML it is standard to have only one body tag throughout our whole application. This original body tag lives inside of our index.html in our public folder. Standard practice is to never use the body tag, just use a div and style accordingly.